win32ss/gdi/ntgdi/region.c

```diff
@@ -2135,6 +2135,7 @@ REGION_bXformRgn(
             {
                 /* Make sure the rect is well-ordered after the xform */
                 RECTL_vMakeWellOrdered(&prgn->Buffer[i]);
+                NT_ASSERT(prgn->Buffer[i].top <= prgn->Buffer[i].bottom);
```

Comment:
This assert is pointless, because that is what the above function will do.

Reply:
I concur with that.